includes/blocks/countdown-timer/render.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+/**
+ * Renders the countdown timer block.
+ *
+ * @since 1.0.0
+ * @return string
+ */
+if ( ! function_exists( 'benenson_render_countdown_timer_block' ) ) {
+	function benenson_render_countdown_timer_block( array $attributes = [] ) {
+		$required = [ 'countdownId', 'title', 'date' ];
+		foreach ( $required as $req ) {
+			if ( ! isset( $attributes[ $req ] ) ) {
+				$attributes[ $req ] = '';
+			}
+		}
+
+		$days           = '0';
+		$hours          = '00';
+		$minutes        = '00';
+		$seconds        = '00';
+		$time_remaining = 0;
+
+		if ( ! empty ( $attributes['date'] ) ) {
+			$start_date     = time();
+			$end_date       = strtotime( $attributes['date'] );
+			$time_remaining = $end_date - $start_date;
+
+			if ( $time_remaining > 0 ) {
+				$days            = intval( $time_remaining / 86400 );
+				$time_remaining %= 86400;
+
+				$hours           = str_pad( intval( $time_remaining / 3600 ), 2, '0', STR_PAD_LEFT );
+				$time_remaining %= 3600;
+
+				$minutes         = str_pad( intval( $time_remaining / 60 ), 2, '0', STR_PAD_LEFT );
+				$time_remaining %= 60;
+
+				$seconds = str_pad( $time_remaining, 2, '0', STR_PAD_LEFT );
+			}
+		}
+
+		$finished         = $time_remaining > 0 ? '' : 'is-finished';
+		$hide_timer       = ! empty ( $attributes['hideTimer'] ) && $attributes['hideTimer'] ? 'data-hide-show=' . $attributes['hideTimer'] . '' : '';
+		$background_color = ! empty ( $attributes['backgroundColor'] ) ? 'background-color:' . $attributes['backgroundColor'] . ';' : '';
+		$background_image = ! empty ( $attributes['backgroundUrl'] ) ? 'background-image: url(' . $attributes['backgroundUrl'] . ');' : '';
+
+		spaceless();
+		?>
+
+		<div class="countdownTimer <?php echo esc_attr( $finished ); ?>" style="<?php echo esc_attr( $background_color ) . esc_attr ( $background_image ); ?>" <?php echo esc_attr( $hide_timer ); ?> data-ref="<?php echo esc_attr( $attributes['countdownId'] ); ?>" data-date="<?php echo esc_attr( $attributes['date'] ); ?>">
+			<?php if ( $time_remaining > 0 ) { ?>
```

Comment:
Use the alternative `if` syntax here:
```php
if ( $condition ) :
    // do stuff
else :
  // do other stuff
endif;
```

Reduce indentation of lines 52-124 by one level.